upgrade/1.3.5/README.md

```diff
@@ -0,0 +1,236 @@
+# CSM 1.3.5 Patch Installation Instructions
+
+## Introduction
+
+This document guides an administrator through the patch update to Cray Systems Management (CSM) `v1.3.5` from `v1.3.0`, `v1.3.1`, `v1.3.2`, `v1.3.3` or `v1.3.4`.
+If upgrading from CSM `v1.2.2` directly to `v1.3.5`, follow the procedures described in [Upgrade CSM](../README.md) instead.
+
+## Bug Fixes and Improvements
+
+* Added monitoring and a `grafana` dashboard for SMF `kafka` server and zookeeper metrics in `prometheus`.
+* Fixed authentication failure with the `keycloak` integration into Nexus due to a change to the `keycloak` `opa` policy which was recently patched.
+* Fixed `dvs-mqtt` error in the Spire server configuration - note this is only an issue for a customer already running `mqtt` in their environment.
+* Upgrade Ceph to `v16.2.13` and stop running local Docker registries on storage nodes.
+
+## Steps
+
+1. [Preparation](#preparation)
+1. [Update `customizations.yaml`](#update-customizationsyaml)
+1. [Setup Nexus](#setup-nexus)
+1. [Upgrade CANU](#upgrade-canu)
+1. [Upgrade services](#upgrade-services)
+1. [Upgrade Ceph and stop local Docker registries](#upgrade-ceph-and-stop-local-docker-registries)
+1. [Verification](#verification)
+1. [Complete upgrade](#complete-upgrade)
+
+## Preparation
+
+1. Start a typescript on `ncn-m001` to capture the commands and output from this procedure.
+
+   ```bash
+   script -af csm-update.$(date +%Y-%m-%d).txt
```

Comment:
date -I (shorter but gives the same output). Just a suggestion for next time.

Reply:
I am going to leave this as is so that it matches the other 1.3 patches. But this is something we should consider for future patches.